### ISSUE TITLE
Keys deletion is not working

### ISSUE THREAD

**Issue by lilessam** [barryvdh/laravel-translation-manager#203]
Whenever I try to delete a key (you use ajax) the request returns 500 error says that `$key variable is not defined`
I figured that in `postDelete` method in the package controller you didn't pass `$key` argument. You should pass it so the key can be deleted successfully.

### That's the solution

```PHP
public function postDelete($group = null, $key)
    {
        if(!in_array($group, $this->manager->getConfig('exclude_groups')) && $this->manager->getConfig('delete_enabled')) {
            Translation::where('group', $group)->where('key', $key)->delete();
            return ['status' => 'ok'];
        }
    }
```

**Comment by Roadirsh** [comment]
This is fixed by the pull request [#191 ](https://github.com/barryvdh/laravel-translation-manager/pull/191) and has been merged.
But @barryvdh should make a new release of it so we can download it easily...

**Comment by barryvdh** [comment]
Is released now as 0.2.9
